pyrsa/rdm/calc.py

```diff
@@ -233,14 +231,12 @@ def calc_rdm_mahalanobis(dataset, descriptor=None, noise=None):
     else:
         measurements, desc, descriptor = _parse_input(dataset, descriptor)
         noise = _check_noise(noise, dataset.n_channel)
-        # calculate difference @ precision @ difference for all pairs
-        # first calculate the difference vectors diff and precision @ diff
-        # then calculate the inner product
-        diff = _calc_pairwise_differences(measurements)
-        diff2 = (noise @ diff.T).T
-        rdm = np.einsum('ij,ij->i', diff, diff2) / measurements.shape[1]
+        kernel = measurements @ noise @ measurements.T
+        rdm = np.expand_dims(np.diag(kernel), 0) + np.expand_dims(np.diag(kernel), 1)\
+            - 2 * kernel
+        rdm = _extract_triu_(rdm) / measurements.shape[1]
         rdm = RDMs(dissimilarities=np.array([rdm]),
-                   dissimilarity_measure='Mahalanobis',
```

Comment:
Shouldn't this be "dissimilarity_measure='Mahalanobis'"?

Reply:
yes it should. Is fixed now